### PR TITLE
Add `blockchain.scripthash.unsubscribe` feature for RPC

### LINKED ISSUE #812: Feature: implement `blockchain.scripthash.unsubscribe` RPC
See https://electrumx-spesmilo.readthedocs.io/en/latest/protocol-methods.html#blockchain-scripthash-unsubscribe

### PR DESCRIPTION
**Pull request by EshaanAgg**
Fixes #812 
[Reference](https://electrumx-spesmilo.readthedocs.io/en/latest/protocol-methods.html#blockchain-scripthash-unsubscribe)

### CODE REVIEW

**Review by romanz — COMMENTED** [review]
Looks good % small comment

**Comment by romanz** [comment]
Thanks for the contribution - please make sure to squash the commits :)

File: src/electrum.rs [hunk 1]
```diff
@@ -312,6 +312,19 @@ impl Rpc {
             .unwrap()
     }
 
+    fn scripthash_unsubscribe(
```
[the remark this answers is not in the document]
Comment (EshaanAgg) [reply src/electrum.rs:315]: Thanks for the input. Will modify the function accordingly!

**Comment by EshaanAgg** [comment]
@romanz Can you squash the commits while merging the PR? Favong some git rebasing errors while squashing.

**Pull request merged**